clientv3/integration/watch_test.go

```diff
@@ -36,9 +36,8 @@ type watcherTest func(*testing.T, *watchctx)
 
 type watchctx struct {
 	clus          *integration.ClusterV3
-	w             clientv3.Watcher
```

Comment:
yeah, that's better. should I remove the `wclient` in watchctx struct? it is not being used anywhere else.